Automattic/jetpack
# Colors: update Primary green reference to match latest brand book

#### Changes proposed in this Pull Request:

The new color is already in use in some places, such as RNA styles. Let's make the switch everywhere.
<img width="332" alt="Screenshot 2021-11-15 at 14 49 24" src="https://user-images.githubusercontent.com/426388/141797322-43cee7d3-b38f-4e8f-80cc-a516f1a1ae8c.png">


#### Jetpack product discussion

﻿Internal references:
- p6TEKc-3tD-p2
- PbJqbW-C-p2

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Check the Jetpack and VaultPress dashboard pages.
* Ensure that the Green matches your expectations.
